pkgs/by-name/aw/awakened-poe/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  appimageTools,
+  makeWrapper,
+  electron,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "awakened-poe";
+  version = "3.25.102";
+
+  src = fetchurl {
+    url = "https://github.com/SnosMe/awakened-poe-trade/releases/download/v${version}/Awakened-PoE-Trade-${version}.AppImage";
+    hash = "sha256-lcdKJ+B8NQmyMsv+76+eeESSrfR/7Mq6svO5VKaoNUY=";
+    name = "${pname}-${version}";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    name = "${pname}-${version}";
+    src = src;
+  };
+
+  dontUnpack = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin $out/share/${pname} $out/share/applications
+
+    cp -a ${appimageContents}/{locales,resources} $out/share/${pname}
+    cp -a ${appimageContents}/awakened-poe-trade.desktop $out/share/applications/${pname}.desktop
```

Why don't we use `awakened-poe-trade`? That's also what it's called in the AUR: https://repology.org/project/awakened-poe-trade/versions
```suggestion
    cp -a ${appimageContents}/{locales,resources} $out/share/awakened-poe
    cp -a ${appimageContents}/awakened-poe-trade.desktop $out/share/applications/awakened-poe.desktop
```